### ISSUE TITLE
History events showing All-service events, for all services

### ISSUE THREAD

**Issue by paulbellamy** [weaveworks/flux#475]
Should only show events which affected this service.

e.g. the history here.
![screen shot 2017-03-02 at 11 26 01](https://cloud.githubusercontent.com/assets/250199/23505322/1347e768-ff3b-11e6-8825-24ab614244ae.png)

**Comment by squaremo** [comment]
Also in the CLI?

**Comment by paulbellamy** [comment]
yes.

**Comment by philwinder** [comment]
I can't recreate this in the cli. Works as expected.
```
 ~/source/gocode/src/github.com/weaveworks/flux   master ●  fluxctl history
TIME                 TYPE  MESSAGE
20 Apr 17 11:47 UTC  v0    Released: dockercloud/hello-world:stdout to no services, by phil, with message "Because I had to"
20 Apr 17 11:30 UTC  v0    Released: quay.io/weaveworks/helloworld:master-9a16ff945b9e to no services, by phil, with message "Because I had to"

 ~/source/gocode/src/github.com/weaveworks/flux   master ●  fluxctl history --service=default/web
TIME                 TYPE  MESSAGE
20 Apr 17 11:47 UTC  v0    Released: dockercloud/hello-world:stdout to no services, by phil, with message "Because I had to"

 ~/source/gocode/src/github.com/weaveworks/flux   master ●  fluxctl history --service=default/helloworld
TIME                 TYPE  MESSAGE
20 Apr 17 11:30 UTC  v0    Released: quay.io/weaveworks/helloworld:master-9a16ff945b9e to no services, by phil, with message "Because I had to"
```

**Comment by philwinder** [comment]
Also seems to work in the UI. Did someone fix it?
![screen shot 2017-04-20 at 12 50 12](https://cloud.githubusercontent.com/assets/8793723/25229439/256c0fb0-25c8-11e7-95f1-389fca0fe8bf.png)

**Comment by philwinder** [comment]
Fixed by #532
